Dockerfile

```diff
@@ -56,11 +56,11 @@ ENV \
   CELLS_WORKING_DIR="/config" \
   GOPATH="/tmp"
 
+COPY --from=buildstage /app/cells /app/
+COPY root/ /
+
 RUN \
+  export PATH=/app/:$PATH && \
```

Comment:
PATH can, and should be set a a part of ENV in the dockerfile, much like [this](https://github.com/linuxserver/docker-baseimage-alpine/blob/c4951e7021be5ba59d1fb93a6a6fbb7a92ba78a2/Dockerfile#L69)